### DISCUSSION TITLE
use IA for up scale definition option (some Fsr or dlls)

### DISCUSSION THREAD

**Discussion by saberbholt** [smplayer-dev/smplayer#788] [Ideas]
Can be a feature for upper definition with the support of IA with Amd Fsr or Nvidia Dlls tecnologies in the future?
I think is posible and can be easy implement with only a one click button or option in the menu, and can begoodin any cases.

The latest versions of Dlls and frs can make a inter frame too or upload the frame rate (for people like see at 60 fps the HD movies)

**Comment by Raza-0007** [comment]
This is a question for the mpv team. smplayer is just a frontend and uses mpv as its engine. mpv provides a way to load thrid-party shaders and scalers to upscale videos on the fly, some of them are AI based neural scalers. [Here is a good list](https://github.com/stax76/awesome-mpv#shaders) and instructions on how to use them.
